treeherder/model/migrations/0009_add_is_try_repo.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-03-09 18:59
```

Comment:
It doesn't matter for this PR, but don't forget to re-run provision on your VM - Django was updated to 1.10.7 in the requirements file a few weeks ago :-)